File: C#/BluffinMuffin.Protocol/Lobby/QuickMode/IdentifyCommand.cs
using BluffinMuffin.Protocol.Enums;
using Com.Ericmas001.Net.Protocol;

namespace BluffinMuffin.Protocol.Lobby.QuickMode
{
    /// <summary>
    /// In Quick mode, you must **identify** yourself. This action puts a unique name to your Tcp Connection for the server. This name will be used as your playername on every table that you will play.
    /// </summary>
    public class IdentifyCommand : AbstractLobbyCommand, ICommandWithResponse<IdentifyResponse>
    {
        /// <summary>
        /// The name you want to have on the server !
        /// </summary>
        [ExampleValue("SpongeBob")]
        public string Name { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <returns></returns>
        public IdentifyResponse Response(bool success, BluffinMessageId msgId, string message)
        {
            return new IdentifyResponse(this) { Success = success, MessageId = msgId, Message = message };
        }
    }
}

File: Python/BluffinMuffin.Protocol/Lobby/QuickMode/IdentifyCommand.py
from AbstractCommand import AbstractCommand

__author__ = 'ericm'
_project_ = 'BluffinMuffin.Protocol'


class IdentifyCommand(AbstractCommand):
    def __init__(self, obj):
        super().__init__('IdentifyCommand', obj)
        self.name = obj['Name']

    def __str__( self ):
        return '{0} ({1})'.format(super().__str__(), self.name)
